src/components/DonationCard.tsx

```diff
@@ -0,0 +1,61 @@
+import { useTranslation } from "next-i18next";
+import colors from "../styles/colors";
+import Image from "next/image";
+
+const DonationCard = () => {
+    const { t } = useTranslation();
+    return (
+        <div
+            style={{
+                width: "clamp(120px, 76vw, 290px)",
```

Comment:
I'll approve this usage, but beware of browser support in the future https://caniuse.com/?search=clamp